src/languages/en-US.json

```diff
@@ -108,9 +109,9 @@
     "GUILD_VOICE" : "voice channel",
     "GROUP_DM" : "group DM",
     "GUILD_CATEGORY" : "channel category",
-    "GUILD_NEWS" : "news channel",
+    "GUILD_NEWS" : "announcement channel",
     "GUILD_STORE" : "store channel",
-    "GUILD_NEWS_THREAD" : "news thread",
+    "GUILD_NEWS_THREAD" : "announcement channel thread",
```

Comment:
"announcement channel thread" having "channel" in it is intentional, though I couldn't find where this is actually used by the bot (the API would need to report a channel having this type).